ExampleApp/log4net.config

```diff
@@ -14,14 +14,15 @@
         <fields value="appdomain,thread,level,class,method,message,date,utcdate,logger,missingproperty,CustomProperty,exception" />
       </layout>
     </appender>
-    <appender name="file" type="log4net.Appender.RollingFileAppender">
-      <file value="myapplog.csv" />
-      <appendToFile value="true" />
-      <rollingStyle value="Size" />
-      <maxSizeRollBackups value="5" />
-      <maximumFileSize value="10MB" />
-      <staticLogFileName value="true" />
-      <layout type="log4net.Csv.CsvLayout, log4net.Csv">
+    <!--<appender name="file" type="log4net.Appender.RollingFileAppender">-->
```

Comment:
Is the change from named xml elements to <param /> elements a required change?  If so, what is it about extending the rolling file appender that requires this?  Or is this a best practice?